src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -26,12 +39,31 @@ export const Search = ({
 }: SearchProps) => {
   const { t } = useTranslation();
 
+  const SearchClearIcon = () => {
+    return (
+      <Tooltip
+        content={t("service-catalog.clear-search", "Clear search")}
+        placement="bottom"
+        size="small"
+      >
+        <StyledIconButton
+          aria-label={t("service-catalog.clear-search")}
```

Comment:
We also need to pass "Clear Search" again as the second parameter, otherwise, until the string is back, we have `service-catalog.clear-search` as a value for the attribute